integration-cli/docker_api_containers_test.go

```diff
@@ -293,7 +293,7 @@ func TestGetContainerStats(t *testing.T) {
 		dec := json.NewDecoder(bytes.NewBuffer(sr.body))
 		var s *stats.Stats
 		// decode only one object from the stream
-		if err := dec.Decode(&s); err != nil {
+		if err := dec.Decode(&s); err != nil && err != io.EOF {
```

Comment:
@LK4D4 It is? I'm getting that error on Fedora. I thought it was "normal" to have a `EOF` error